easybuild/easyconfigs/v/VBZ_Compression/VBZ_Compression-1.0.1-gompi-2020b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'VBZ_Compression'
```

Comment:
I hate to be pedantic, but I would strongly prefer `VBZ-Compression` over `VBZ_Compression`...

Reply:
done